src/trousse/feature_fix.py

```diff
@@ -34,48 +32,73 @@ def convert_maps_from_tuple_to_str(group_id_to_tuple_map):
 
 
 def split_continuous_column_into_bins(
-    df_info: DataFrameWithInfo, col_name, bin_threshold
+    df_info: DataFrameWithInfo,
+    col_name,
+    bin_thresholds,
+    extra_padding_ratio: float = 0.4,
 ):
     """
-    This function adds a column to DataFrame df_info called "[col_name]_bin_id" where we split the "col_name" into bins
-    :param df_info: DataFrameWithInfo -> DataFrameWithInfo instance containing the 'col_name' column to split
-    :param col_name: String -> Name of the column to be split into discrete intervals
-    :param bin_threshold: List -> It contains the thresholds used to separate different groups
-                                  (the threshold will be included in the bin with higher values)
-    :return: pd.DataFrame -> Same "df_info" passed with a new column with the bin_indices
-                             which the column value belongs to
-             Dict[List] -> Dictionary with the bin_indices as keys and bin_ranges as values
+    Split the continuous values from ``df_info`` column ``col_name`` into discrete bins.
+
+    called "[col_name]_bin_id" where we split the "col_name" into bins
+
+    Parameters
+    ----------
+    df_info : DataFrameWithInfo
+        DataFrameWithInfo instance containing the 'col_name' column to split
+    col_name : str
+        Name of the column to be split into discrete intervals
+    bin_thresholds : List
+        It contains the thresholds used to separate different groups
+        (the threshold will be included in the bin with higher values)
```

Comment:
do you mean that the samples with values corresponding to the threshold will be included.. where?